tests/conftest.py

```diff
@@ -27,6 +29,37 @@ def temp_solcx_path(monkeypatch):
         shutil.rmtree(solcx_install_path, ignore_errors=True)
 
 
+@pytest.fixture(
+    scope="session",
+    autouse=os.environ.get("APE_SOLIDITY_USE_SYSTEM_SOLC") is None,
+)
+def setup_session_solcx_path(request):
+    """
+    Creates a new, temporary installation path for solcx when the test suite is
+    run.
+
+    This ensures the Solidity installations do not conflict with the user's
+    installed versions and that the installations from the tests are cleaned up
+    after the suite is finished.
+    """
+    from _pytest.monkeypatch import MonkeyPatch
```

Comment:
I think there is a monkeypatch fixture, or no?

Reply:
@unparalleled-js it can't be used in non-function scopes unfortunately, so you have to instantiate it manually. Doesn't look like the proposed `monkeysession` is in pytest yet either

https://github.com/pytest-dev/pytest/issues/363